external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/DirectKafkaInputDStream.scala

```diff
@@ -222,9 +222,10 @@ private[spark] class DirectKafkaInputDStream[K, V](
     val description = offsetRanges.filter { offsetRange =>
       // Don't display empty ranges.
       offsetRange.fromOffset != offsetRange.untilOffset
-    }.map { offsetRange =>
+    }.toSeq.sortBy(-_.count()).map { offsetRange =>
```

Comment:
I think it's OK to sort, but will it be less surprising to sort by topic name?

Reply:
thank you for reviewing, it seems to be reasonable to sort by topic(or partition) name, but it's useful to sort by count when inspecting your data skewness problem, and it will help you to have an intuitive acknowledge of some data-intensive topics.

Reply:
Sure, but the topics might also change order on every load or something. I don't feel strongly about it. Any sort order is probably more useful than arbitrary.

Reply:
Any sort order is useful, and sorting by count may provide the most important information we need. So I think it's better. Can someone else provide some suggestions?

Reply:
@dengziming . Could you try this with Apache Spark `minPartitions` feature? Actually, Apache Spark has a feature to handle the topic skewness. The skewness seems not important in that case.

Reply:
@dongjoon-hyun, thank you, I will try it. `minPartitions` is a parameter of `sparkContext.textFile` method, how can I use it when reading from Kafka?

Reply:
Please take a look at our official Kafka integration document. `minPartitions` is one of our typical parameters like `startingOffsets` and `endingOffsets`.
- https://spark.apache.org/docs/latest/structured-streaming-kafka-integration.html#creating-a-kafka-source-for-batch-queries